packages/desktop/components/NetworkCard.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+    import { localize } from '@core/i18n'
+    import { NetworkConfigRoute, networkConfigRouter } from '@desktop/routers'
+    import { ClickableTile, Text, Icon, FontWeight, TextType, NetworkIcon, NetworkStatusPill } from '@ui'
+    import { Icon as IconEnum } from '@auxiliary/icon'
+    import { truncateString } from '@core/utils'
+    import { NetworkHealth, NetworkId } from '@core/network'
+
+    export let name: string
+    export let address: string
+    export let status: NetworkHealth
+
+    function onTileClick(): void {
+        $networkConfigRouter.goTo(NetworkConfigRoute.ChainInformation)
+    }
+
+    function onQrCodeIconClick(): void {
+        $networkConfigRouter.goTo(NetworkConfigRoute.ChainDepositAddress)
+    }
+</script>
+
+<ClickableTile classes="bg-transparent border border-solid border-gray-200" onClick={onTileClick}>
+    <div class="w-full flex flex-col gap-5">
+        <div class="flex flex-row justify-between items-center">
+            <div class="flex flex-row gap-2 items-center">
+                <NetworkIcon networkId={NetworkId.Shimmer} />
```

Comment:
When I hover over the hard, there's a tiny white outline around this icon :thinking: